VideoLocker/res/values/colors.xml

```diff
@@ -160,5 +160,5 @@
     <color name="edx_utility_warning">#FDBC56</color>
     <color name="edx_utility_error">#B20610</color>
 
-
+    <drawable name="image_loading_placeholder">@color/edx_grayscale_neutral_base</drawable>

```

Comment:
This should probably be a theme attribute, instead of an alias drawable in colors.xml.